regulations/static/regulations/css/scss/partials/_header.scss

```diff
@@ -107,7 +107,7 @@
   position: fixed;
   top: $mainhead_height;
   min-width: 100%;
-  background: $panel_background_color;
+  background-color: $panel_background_color;
   color: $black;
```

Comment:
The font color should be `$atf_sub_head_background`, what was the background color. I think that is declared here but not sure.

Reply:
That'll be an ATF-specific override. The default sub-head is black(ish) on white(ish), as in the screenshots.